app/src/main/java/com/asfoundation/wallet/referrals/ReferralInteractor.kt

```diff
@@ -26,17 +26,6 @@ class ReferralInteractor(
         }
```

Comment:
Single line expression should be used here => .map { hasDifferentInformation(friendsInvited.toString() + isVerified, it) }